CHANGES_NEXT_RELEASE

```diff
@@ -6,3 +6,4 @@
 - [cygnus-ngsi][hardening] Pre-aggregate batches in NGSISTHSink (#1359)
```

Comment:
I've taken the following decission: since the usage of file channels is related to Cygnus reliability, I'll introduce (in a new PR) a documentation topic regarding that. Of course, I'll not speak only about file channels, but I'll introduce persistence retries, traffic absortion peaks... even, I'll talk about HA. That's because I think it fits better in a new different PR. What do you think, @fgalan ?